anthos-baremetal-edge-deployment/acm-config-sink/namespaces/pos/api-server-lb.yaml

```diff
@@ -12,6 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# [START anthosbaremetal_pos_api_server_lb_service_api_server_lb]
```

Comment:
Sorry if I missed any previous conversations. But is this region tag formed following some convention/guidance? This looks a bit too long.

Reply:
For our sample app repos, we have this format:
```
REGION TAG FORMAT: YAML  (with metadata name)
<product-prefix>_<directory-with-underscores>_<filename-without-extension>_<kind>_<metadata.name>
REGION TAG FORMAT: YAML  (w/out metadata name)
<product-prefix>_<parent-directory-with-underscores>_<directory-with-underscores>_<filename-without-extension>_<kind>
REGION TAG FORMAT: SHELL
<product-prefix>_<directory-with-underscores>_<filename-without-extension>
```
It's to make our lives easier (I'm not about to go through every yaml file 😅 )

I know some of these region tags are non-descriptive, but with our sample app repos these are just to get it into sample browser (it's not as high visibility as your efforts with nodejs-container repo :) )

that being said, the approach I took with boa and microservices-demo was to edit their `make-release.sh` ([example](https://github.com/GoogleCloudPlatform/bank-of-anthos/blob/6bc21926ad2eb48d7a591a9e0ebd5ca81732d5e0/release/make-release.sh#L55)), and add region tags to the release yaml files. If that's the approach we can take in this repo, I can follow up with you!

Reply:
Thank you for the detailed explanation. I understand it now. For sure we might want to have a similar process to what we see in the other samples app once we have this formally launched.